### PR TITLE
Small improvements to on-premise setup

### PR DESCRIPTION
**Pull request by AntiD2ta**
## Changes:

- Updated container_name fields of docker-compose scripts templates for cleaner names
- Removed restarts from validator services
- Refactored code
- Display docker containers after running the docker-compose script (Displaying only containers belonging to the script is in backlog)
- Upgraded error checking in the pipeline setup
- Run only execution and consensus services when running the docker-compose script
- `1click cli` runs with `-r -i --run` flags by default
- Added `1click logs` command to show the running docker-compose script services
- Added validator import service on those clients that supported it
- Show dependencies install instructions and docker-compose script executing instructions by default
- Generate config file by default with `1click cli`
- Ensure Docker engine is running before running the docker-compose script
- Minor fixes

### CODE REVIEW

File: cmd/cli.go [hunk 1]
```diff
@@ -121,24 +126,21 @@ func init() {
...
-		Items: []string{optShow, optInstall, optExit},
+		Items: []string{optInstall, optExit},
 	}
 
+	if err = utils.HandleInstructions(pending, utils.ShowInstructions); err != nil {
+		return fmt.Errorf(configs.ShowingInstructionsError, err)
+	}
 	_, result, err := prompt.Run()
 	if err != nil {
 		return fmt.Errorf("prompt failed %s", err)
```
Comment (1swaraj) [inline cmd/cli.go:140]: No hardcoded errors please

File: internal/utils/cmd_execution.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+	errChans := make([]<-chan error, 0)
+	errChans = append(errChans, goCopy(&wait, stdin, &scriptBuffer, true))
+	if script.GetOutput {
+		cmd.Stdout = &combinedOut
+		cmd.Stderr = &combinedOut
+	} else {
+		errChans = append(errChans, goCopy(&wait, os.Stdout, stdout, false))
+		errChans = append(errChans, goCopy(&wait, os.Stderr, stderr, false))
+	}
```
Comment (1swaraj) [inline internal/utils/cmd_execution.go:95]: Can you add comments for this part ?
Confusing

**Review by 1swaraj — COMMENTED** [review]
What is the plan for keystore setup etc ?
Also can we write detailed comments on complex functions ?

File: internal/pkg/generate/generate_config.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+params :-
+a. path string
+Path of generated config file
+
+returns :-
+a. error
+Error if any
+*/
+func GenerateConfig(path string) (err error) {
+	rawTmp, err := templates.Config.ReadFile("config/config.tmpl")
```
[the remark this answers is not in the document]
Comment (AntiD2ta) [reply internal/pkg/generate/generate_config.go:25]: They are all in the templates folder, organized by scopes (`conf` for configuration templates, `envs` for .env templates, `services` for docker-compose templates, and `setup` for installation scripts templates)

**Pull request merged**